vms/platformvm/camino_vm_test.go

```diff
@@ -874,7 +937,7 @@ func TestExcludeMemberProposals(t *testing.T) {
 			addrStateTx, err := vm.txBuilder.NewAddressStateTx(
```

Comment:
> give root admin **ConsortiumSecretary** role
Update also the role name in the comment?

Reply:
done, also updated similar places